appunite/Loudius
# [LD-154] Add error message to action_finished event.

# Pull-Request
## Description

### Why?

We want to track the error message, because it will help trace the root of the issue.

### What?

Added `error_message` parameter in `action_finished` event, when success is false.

## Links to related issues

- Fixes [LD-154](https://loudius.atlassian.net/browse/LD-154)

## Demo


Nothing to show :)

## How to test

You can open the application, turn off the internet, e.g. click notify and check if event `action_finished` with `error_message` parameter exists.

## Documentation
- Link to github comment, where it was proposed: [comment](https://github.com/appunite/Loudius/pull/180#discussion_r1423648695)

## Checklist


- [x] Functionality is covered by unit tests
- [x] Functionality is covered by integration tests
